content/applications/finance/accounting/others/analytic/analytic_accounting.rst

```diff
@@ -0,0 +1,122 @@
+- :guilabel:`Domain`: choose where your plan is applicable;
+- :guilabel:`Financial Accounts Prefix`: select the prefix of the account(s) to which this plan
+  should be applied;
+- :guilabel:`Product Category`: decide to which product category the plan applies;
+- :guilabel:`Applicability`: decide how your plan behaves in the widget when creating a new journal
+  entry:
+
+  - :guilabel:`Optional`: if selected, it is not mandatory to add the analytic plan in the widget;
+  - :guilabel:`Mandatory`: if selected, an orange bullet is visible in the widget next to the plan
+    until the analytic distribution is done (the bullet then turns to green); it is not possible to
+    confirm the entry if no analytic account is selected;
+  - :guilabel:`Unavailable`: if selected, the plan is not available in the widget.
+
+Two smart buttons are available in the top-right corner:
+
+   - :guilabel:`Subplans`: can be created to have a more complex analytic structure. Click the
+     :guilabel:`Subplans` smart button, and then :guilabel:`New` to add a subplan;
+   - :guilabel:`Analytic Accounts`: to reach the subplans related to the plan.
+
+.. note::
+   - The analytic widget is prefilled based on the :guilabel:`Applicability`, and the
+     :ref:`Analytic Distribution Models <analytic_distribution_models>`;
+   - Each :guilabel:`Analytic plan` must have at least one :guilabel:`Analytic Account`.
+
+Analytic distribution
+=====================
+
+When creating an invoice or a bill it is mandatory to add a plan in the
+:guilabel:`Analytic Distribution` column. This opens a **widget**, in which you can fill in the
+different information. You can add **tags** to reflect the related :guilabel:`Analytic Accounts`,
+and decide how to split the costs between the accounts by modifying the percentage.
+
+.. image:: analytic_accounting/analytic_distribution.png
+   :align: center
+   :alt: create a distribution template
+
+.. _analytic_distribution_models:
+
+Analytic distribution models
+----------------------------
+
+The :guilabel:`Analytic Distribution Models` automatically apply a specific distribution based on
+defined criteria.
+
+To create a new :guilabel:`Analytic Distribution Model`, go to :menuselection:`Accounting -->
+Configuration --> Analytic Distribution Models`, click :guilabel:`New` and set the conditions your
+model has to meet to automatically apply:
+
+- :guilabel:`Partner`: select a partner for which the analytic distribution will be used;
+- :guilabel:`Partner Category`: select a partner category for which the analytic distribution will
+  be used;
+- :guilabel:`Accounts Prefix`: this analytic distribution will apply to all financial accounts
+  sharing the prefix specified;
+- :guilabel:`Product`: select a product for which the analytic distribution will be used;
+- :guilabel:`Product Category`: select a product category for which the analytic distribution will
+  be used;
+  :guilabel:`Analytic Account` specified in analytic default;
```

Comment:
There is only one column named Analytic. Also, that's where you specify the distribution between different accounts, no? Might be another recent change